src/Adapter/Driver/Mysqli/Connection.php

```diff
@@ -146,7 +146,7 @@ public function connect()
             }
         }
 
-        $this->resource->real_connect($hostname, $username, $password, $database, $port, $socket, $flags);
+        @$this->resource->real_connect($hostname, $username, $password, $database, $port, $socket, $flags);
```

Comment:
@machek we are not using the @ operator in Zend Framework code base. Can you manage this using try catch?